aws/terraform/modules/elastic-file-system/README.md

```diff
@@ -0,0 +1,20 @@
+# Elastic File System Module
+
+This module creates an Elastic Files System (EFS) in AWS, as well as necessary network resources to allow access to the file system from your cluster. It also provisions an EFS-backed Kubernetes StorageClass. This is used by `privacy-api` from Privacy Dynamics, which requires persistent storage that can be used simultaneously across multiple pods and nodes.
```

Comment:
Typo (`File(s)`).
Let's restate the purpose a bit to mention that the data is temporary and avoid any specific pod name mentions (these could change in the future). 

```suggestion
This module creates a file system in [Amazon Elastic File System (EFS)](https://docs.aws.amazon.com/efs/latest/ug/whatisefs.html), as well as necessary network resources to allow access from your cluster. It also provisions an EFS-backed Kubernetes StorageClass. This is used by the Privacy Dynamics application to store temporary data that can be used simultaneously from multiple pods and nodes.
```